README.md

```diff
@@ -32,6 +32,7 @@ This extension connects [coc.nvim][] to the [clangd][] language server.
 - `clangd.disableDiagnostics`: disable diagnostics from clangd, default `false`
 - `clangd.path`: path to `clangd` executable, default `clangd`
 - `clangd.arguments`: arguments for `clangd` server, default `[]`
+- `clangd.fallbackFlags`: flags used when no specific compile commands found, default `[]`
```

Comment:
suggest "used to parse files" - just to hint that these are clang flags not clangd flags

Reply:
You means the description? `flags used to parse files when no specific compile commands found`, is this ok?

Reply:
Exactly, thanks!